testing/coreruleset/coreruleset_test.go

```diff
@@ -191,7 +194,16 @@ SecRule REQUEST_HEADERS:X-CRS-Test "@rx ^.*$" \
 	}
 
 	s := httptest.NewServer(txhttp.WrapHandler(waf, t.Logf, http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "hello!")
+		// Emulated httpbin behaviour: /anything endpoint acts as an echo server, writing back the request body
+		if r.URL.Path == "/anything" {
```

Comment:
shall we call it `/echo` instead? to make it easier to identify what it is about?

Reply:
It would make it easier, but CRS tests expect to have the [httpbin as the backend](https://coreruleset.org/docs/development/testing/) e.g. [953120-3 test uri is `/anything`](https://github.com/coreruleset/coreruleset/blob/v4.0/dev/tests/regression/tests/RESPONSE-953-DATA-LEAKAGES-PHP/953120.yaml#L84). I think that this handler has to mimic as much as possible the `httpbin` behavior.